MadDonkeySoftware/corp-hq
# Added ability for sub jobs. Added container source isolation.

### Story
N/A

### Summary
* Added message levels to messages emitted by jobs running
* Added ability for jobs to spawn off sub jobs.
* Created new `manager` to handle moving jobs from the DB the queue
  * This project also handles splitting jobs into smaller jobs via the `Adjudication` namespace
* Updated the `quick-start.sh` and supporting artifacts to start containers with code isolation for the projects requiring compilation.
* Added solution file in prep. for demoing this code at work
* Added some stub work for the UI around starting a mining operation.

### Testing
* Create a new job and get it's details
* Run the app and verify log in / log out
* Run the behave tests.
